ibm/service/power/data_source_ibm_pi_network_security_groups.go

```diff
@@ -0,0 +1,266 @@
+// Copyright IBM Corp. 2024 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package power
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
```

Comment:
could probably merge imports here

Reply:
What do you mean by merging imports?

Reply:
just squashing them together, but it's not really important